packages/lesswrong/lib/collections/posts/schema.ts

```diff
@@ -494,6 +495,18 @@ const schema = {
     }),
     canRead: ['guests'],
   },
+
+  // tagRelation
+  tagRel: resolverOnlyField({
+    type: "TagRel",
+    graphQLtype: "TagRel",
+    viewableBy: ['guests'],
+    graphqlArguments: 'tagId: String',
+    resolver: (post, {tagId}, context) => {
+      const tagRel = TagRels.findOne({tagId: tagId, postId: post._id})
```

Comment:
This is an N+1 select. To get decent performance, it needs to be using the loaders system.